public/samples/DataStreams/StreamsUpkeep.sol

```diff
@@ -75,28 +63,28 @@ contract StreamsUpkeep is ILogAutomation, StreamsLookupCompatibleInterface {
         address quoteAddress;
     }
 
-    event ReportVerified(BasicReport indexed report);
-    event PriceUpdate(int192 price);
+    event PriceUpdate(int192 indexed price);
 
-    address public immutable FEE_ADDRESS;
+    IVerifierProxy public verifier;
+
+    address public FEE_ADDRESS;
     string public constant STRING_DATASTREAMS_FEEDLABEL = "feedIDs";
     string public constant STRING_DATASTREAMS_QUERYLABEL = "timestamp";
-    string[] public feedsHex = [
-        "0x00023496426b520583ae20a66d80484e0fc18544866a5b0bfee15ec771963274"
+    string[] public feedIds = [
+        "0x00029584363bcf642315133c335b3646513c20f049602fc7d933be0d3f6360d3" // Ex. Basic ETH/USD price report
```

Comment:
May be helpful to put the Stream ID url in the comment?
https://docs.chain.link/data-streams/stream-ids

Reply:
Done